### PR TITLE
finished e2e tests for order history page

### PR DESCRIPTION
**Pull request by kapoliub**
finished e2e test for empty order history page

### CODE REVIEW

File: cypress/integration/order-history.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+describe('empty order history test', () => {
+  const MOCK_EMAIL = 'kapov@gmail.com';
+  const MOCK_PASSWORD = 'QazxsW21';
+  it('should enter and open order history page', () => {
+    cy.visit('/login');
+    cy.get(
+      '.makeStyles-emailInput-487 > .MuiInputBase-root > .MuiInputBase-input'
```
Comment (TorskyM) [inline cypress/integration/order-history.test.js:7]: Class names created by `makeStyles`  are not static, use the `data-cy` attribute to select element

File: cypress/integration/order-history.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    )
+      .should('be.visible')
+      .type(MOCK_EMAIL);
+    cy.get(
+      '.makeStyles-passwordInput-486 > .MuiInputBase-root > .MuiInputBase-input'
```
Comment (TorskyM) [inline cypress/integration/order-history.test.js:12]: Class names created by `makeStyles` are not static, use the `data-cy` attribute to select element

**Review by TorskyM — COMMENTED** [review]
Change pull request title to `Order history page e2e tests`

**Review by ahd3r — APPROVED** [review]

File: cypress/integration/order-history.test.js (hunk 1 above)
Comment (kapoliub) [reply cypress/integration/order-history.test.js]: this problem has been solved

File: cypress/integration/order-history.test.js (hunk 2 above)
Comment (kapoliub) [reply cypress/integration/order-history.test.js]: this problem has been solved

**Review by TorskyM — APPROVED** [review]

**Review by maksym-strus — APPROVED** [review]

**Pull request merged**